app/middlewares/auth-jwt.js

```diff
@@ -2,21 +2,96 @@ const jwt = require('jsonwebtoken');
 const config = require('../config/auth.config');
 const {user: User, role: Role} = require('../models/index');
 
-// Verify the JWT token provided in the request headers
-exports.verifyToken = (req, res, next) => {
-  let token = req.headers['x-access-token'];
+exports.accessToken = async (payloadData) => {
+  try {
+    const JWT_ACCESS_TOKEN_SECRET = config.JWT_ACCESS_TOKEN_SECRET;
+    if (!JWT_ACCESS_TOKEN_SECRET) {
+      console.log(`Unable to process Constant [JWT_ACCESS_TOKEN_SECRET]`);
+      return null;
+    }
+    const jwtSecretKey = JWT_ACCESS_TOKEN_SECRET;
+    const jwtConfigOptions = {
+      expiresIn: config.JWT_ACCESS_EXPIRES_IN,
+      algorithm: config.ALGORITHM,
+    };
+    const jwtToken = await jwt.sign(
+      payloadData,
+      jwtSecretKey,
+      jwtConfigOptions
+    );
+    return jwtToken;
+  } catch (error) {
+    console.log(error);
+  }
+};
 
-  if (!token) {
-    return res.status(403).send({message: 'No token provided!'});
+exports.refreshToken = async (payloadData) => {
+  try {
+    const JWT_REFRESH_TOKEN_SECRET = config.JWT_REFRESH_TOKEN_SECRET;
+    if (!JWT_REFRESH_TOKEN_SECRET) {
+      console.log(`Unable to process Constant [JWT_REFRESH_TOKEN_SECRET]`);
+      return null;
+    }
+    const jwtSecretKey = JWT_REFRESH_TOKEN_SECRET;
+    const jwtConfigOptions = {
+      expiresIn: config.JWT_REFRESH_EXPIRES_IN,
+      algorithm: config.ALGORITHM,
+    };
+    const refreshToken = await jwt.sign(
+      payloadData,
+      jwtSecretKey,
+      jwtConfigOptions
+    );
+    return refreshToken;
+  } catch (error) {
+    console.log(error);
```

Comment:
console.error

Reply:
Is it okay if I convert all the console.log into console.error or should I do them only on the lines you've requested?

Reply:
Okay okay, I'll keep them in mind and change accordingly